recipe/meta.yaml

```diff
@@ -1,48 +1,45 @@
 {% set name = "libnvjitlink" %}
-{% set version = "12.4.127" %}
-{% set cuda_version = "12.4" %}
+{% set version = "12.8.93" %}
+{% set cuda_version = "12.8" %}
+{% set platform = "placeholder" %}  # [osx]
 {% set platform = "linux-x86_64" %}  # [linux64]
-{% set platform = "linux-ppc64le" %}  # [ppc64le]
 {% set platform = "linux-sbsa" %}  # [aarch64]
 {% set platform = "windows-x86_64" %}  # [win]
 {% set target_name = "x86_64-linux" %}  # [linux64]
-{% set target_name = "ppc64le-linux" %}  # [ppc64le]
 {% set target_name = "sbsa-linux" %}  # [aarch64]
 {% set target_name = "x64" %}  # [win]
 {% set extension = "tar.xz" %}  # [not win]
 {% set extension = "zip" %}  # [win]
 
 package:
-  name: {{ name|lower }}
+  name: {{ name|lower }}-split
   version: {{ version }}
 
 source:
   url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
-  sha256: 0e0ec59ee56d3dfa29c66bd4e225b1a6330d42b36545ab4377880009d42b675c  # [linux64]
-  sha256: a08da1a0b990ef7eb845655eb51ddc9fc920921da14ee2b32ad8daaebeb1c68b  # [ppc64le]
-  sha256: c85f5c50f18e1b8249a318432dbefdbb752ec81d374a59ab452f8d74acaf6017  # [aarch64]
-  sha256: d8f4086215f482263dbfbe47a3580e88acbcdacbb284f8aa0c21a8fe408e671d  # [win]
+  sha256: 29220664ad272c513d3213133ace6d6590303e46b42a17b05bc2bee90e54a888  # [linux64]
+  sha256: d5be4fc9b26e07a687249f8a5151fa093f512f1c50a629d52e659ee9ce984aca  # [aarch64]
+  sha256: 5680b0a42ddf20f11705ca9c365d002f032ab876d3fe44382eeba633b558ccc0  # [win]
 
 build:
   number: 1
   binary_relocation: false
   skip: true  # [osx]
 
-test:
-  requires:
-    - patchelf  # [linux]
-  files:
-    - test-rpath.sh
-  commands:
-    - test -L $PREFIX/lib/libnvJitLink.so.{{ version.split(".")[0] }}                            # [linux]
-    - test -L $PREFIX/lib/libnvJitLink.so.{{ version }}                                          # [linux]
-    - test -L $PREFIX/targets/{{ target_name }}/lib/libnvJitLink.so.{{ version.split(".")[0] }}  # [linux]
-    - test -f $PREFIX/targets/{{ target_name }}/lib/libnvJitLink.so.{{ version }}                # [linux]
-    - bash test-rpath.sh                                                                         # [linux]
-    - if not exist %LIBRARY_BIN%\nvJitLink*.dll exit 1                                           # [win]
+requirements:
+  build:
+    - cf-nvidia-tools 1.*  # [linux] 
+    - patchelf <0.18.0     # [linux]
 
 outputs:
   - name: libnvjitlink
+    build:
+      binary_relocation: false
```

Comment:
```suggestion
```
It's not there on [conda-forge](https://github.com/conda-forge/libnvjitlink-feedstock/blob/d5ad84fd2b8ced535d4f5976b5d993a2aca5c5b1/recipe/meta.yaml#L37), although perhaps this is just cosmetic, since it's specified at the top-level, but with multi-output feedstocks you never know?

Reply:
The generic `binary_relocation: false` seems redundant. The output's one is important. If I added it here then there was a failure of the output.